projects/plugins/jetpack/modules/custom-post-types/comics.php

```diff
@@ -288,71 +373,124 @@ public function manage_posts_custom_column( $column_name, $post_ID ) {
+				)
+				: false,
+			6  => sprintf(
+				/* Translators: link to comic item's page. */
+				__( 'Comic published. <a href="%s">View comic</a>', 'jetpack' ),
+				esc_url( get_permalink( $post->ID ) )
+			),
 			7  => esc_html__( 'Comic saved.', 'jetpack' ),
-			8  => sprintf( __( 'Comic submitted. <a target="_blank" href="%s">Preview comic</a>', 'jetpack'), esc_url( add_query_arg( 'preview', 'true', get_permalink( $post->ID ) ) ) ),
-			9  => sprintf( __( 'Comic scheduled for: <strong>%1$s</strong>. <a target="_blank" href="%2$s">Preview comic</a>', 'jetpack' ),
-			// translators: Publish box date format, see https://php.net/date
-			date_i18n( __( 'M j, Y @ G:i', 'jetpack' ), strtotime( $post->post_date ) ), esc_url( get_permalink($post->ID) ) ),
-			10 => sprintf( __( 'Comic draft updated. <a target="_blank" href="%s">Preview comic</a>', 'jetpack' ), esc_url( add_query_arg( 'preview', 'true', get_permalink( $post->ID ) ) ) ),
+			8  => sprintf(
+				/* Translators: link to portfolio item's page. */
+				__( 'Comic submitted. <a target="_blank" href="%s">Preview comic</a>', 'jetpack' ),
+				esc_url( add_query_arg( 'preview', 'true', get_permalink( $post->ID ) ) )
+			),
+			9  => sprintf(
+				/* Translators: link to comic item's page. */
+				__( 'Comic scheduled for: <strong>%1$s</strong>. <a target="_blank" href="%2$s">Preview comic</a>', 'jetpack' ),
+				// translators: Publish box date format, see https://php.net/date
+				date_i18n( __( 'M j, Y @ G:i', 'jetpack' ), strtotime( $post->post_date ) ),
+				esc_url( get_permalink( $post->ID ) )
+			),
+			10 => sprintf(
+				/* Translators: link to comic item's page. */
+				__( 'Comic draft updated. <a target="_blank" href="%s">Preview comic</a>', 'jetpack' ),
+				esc_url( add_query_arg( 'preview', 'true', get_permalink( $post->ID ) ) )
+			),
 		);
 
 		return $messages;
 	}
 
 	/**
 	 * Should this Custom Post Type be made available?
+	 *
+	 * @return bool
 	 */
 	public function site_supports_comics() {
-		/**
-		 * @todo: Extract this out into a wpcom only file.
-		 */
-		if ( 'blog-rss.php' == substr( $_SERVER['PHP_SELF'], -12 ) && count( $_SERVER['argv'] ) > 1 ) {
-			// blog-rss.php isn't run in the context of the target blog when the init action fires,
-			// so check manually whether the target blog supports comics.
-			switch_to_blog( $_SERVER['argv'][1] );
-			// The add_theme_support( 'jetpack-comic' ) won't fire on switch_to_blog, so check for Panel manually.
-			$supports_comics = ( ( function_exists( 'site_vertical' ) && 'comics' == site_vertical() )
-								|| current_theme_supports( self::POST_TYPE )
-								|| get_stylesheet() == 'pub/panel' );
-			restore_current_blog();
-
-			/** This action is documented in modules/custom-post-types/nova.php */
-			return (bool) apply_filters( 'jetpack_enable_cpt', $supports_comics, self::POST_TYPE );
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
I note this is a potential behavior change. OTOH, I don't see a "blog-rss.php" in wporg or calypso. 🤷

Reply:
I had assumed only WordPress.com would store blog IDs there, since this was created for WordPress.com simple in the first place. 
The `blog-rss.php` script seems to stil exist on WordPress.com at the moment, although I'm not sure it's still in use.